Makefile

```diff
@@ -1,6 +1,7 @@
 BUILD_DIR=$(shell pwd)/build
 BIN_DIR=$(BUILD_DIR)/bin
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
+YUM_WRAPPER=$(shell base64 -w 0 scripts/yum-wrapper)
```

Comment:
ah, was wondering about this... wondered if livecd-creator created environment variables to be lowercase. awfully distracted at the moment... :-s